### PR TITLE
@looker/components-theme-editor package + story updates

### PR DESCRIPTION
**Pull request by lukelooker**
Leverage new ThemeEditor components for Storybook examples

### :sparkles: Changes

- New @looker/components-theme-editor package. Per package README:

>This package publishes several components that can be used to build ThemeEditor interfaces for @looker/components-based applications. These are currently used for proof-of-concept implementations in our Storybook and documentation sites.
>
>  These are not intended for use in production applications and may have experience radical API or UI changes in the future.

- Leverage new ThemeEditor components within existing Storybook examples

### :white_check_mark: Requirements

- [ ] Includes test coverage for all changes
- [x] Build and tests are passing
- [ ] Update documentation
- [x] Updated CHANGELOG
- [ ] Checked for i18n impacts
- [ ] Checked for a11y impacts
- [ ] PR is ideally < ~400LOC

### CODE REVIEW

File: packages/components-theme-editor/src/utils/filterObject.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+export const filterObject = (full: { [key: string]: any }, allowed: string[]) =>
+  Object.keys(full)
+    .filter((key) => allowed.includes(key))
+    .reduce((obj, key) => {
+      return {
+        ...obj,
+        [key]: full[key],
+      }
+    }, {})
```
Comment (dbchristopher) [inline packages/components-theme-editor/src/utils/filterObject.ts:35]: Is this different than [lodash's `pick`](https://lodash.com/docs/4.17.15#pick)?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Preview: `InputFilters` component and tests (this component is not yet ready for general-use)
 - Preview: `ActionListControls` component (this component is not yet ready for general-use)
+- Experimental: `@looker/components-thmee-editor` package
```
Comment (dbchristopher) [inline CHANGELOG.md:12]:
```suggestion
- Experimental: `@looker/components-theme-editor` package
```

**Review by dbchristopher — DISMISSED** [review]
Seems good to me! Just caught one spelling issue in Changelog, and the `filterObject` helper might not be necessary 🤷‍♂️ But no major blockers!

File: packages/components-theme-editor/src/utils/filterObject.ts (hunk 1 above)
Comment (lukelooker) [reply packages/components-theme-editor/src/utils/filterObject.ts:35]: 🤦

Comment (lukelooker) [reply packages/components-theme-editor/src/utils/filterObject.ts:35]: It apparently is also _never used_.

**Review by dbchristopher — DISMISSED** [review]

**Review by dbchristopher — APPROVED** [review]

**Pull request merged**